ci/test_python_cudf.sh

```diff
@@ -14,6 +14,15 @@ EXITCODE=0
 trap "EXITCODE=1" ERR
 set +e
 
+rapids-logger "pytest pylibcudf"
+pushd python/cudf/cudf/pylibcudf_tests
+python -m pytest \
+  --cache-clear \
+  --numprocesses=8 \
```

Comment:
*suggestion*: I would start without xdist, since there are few tests, running "serially" is probably faster overall.